Fix:

./lft_types.h:227:5: error: type name requires a specifier or qualifier
  227 |     SLIST_ENTRY(trace_packet_info_s) next_by_hop;
      |     ^
--- lft_types.h.orig	2020-04-09 15:46:51.000000000 -0500
+++ lft_types.h	2025-09-16 23:00:25.000000000 -0500
@@ -120,9 +120,9 @@
 #include "lft_lsrr.h"
 #include "whois.h"
 
-#if defined(__FreeBSD__)
+#if defined(__FreeBSD__) || defined(DARWIN)
 #include <sys/queue.h>
-#elif !defined(DARWIN) && !defined(NETBSD)
+#elif !defined(NETBSD)
 #include "lft_queue.h"
 #endif
 
